### ISSUE TITLE
Plugins page giving sub-plugins leading to multiple boxes having the same name

### ISSUE THREAD

**Issue by shrutimantri** [kestra-io/docs#1456]
### Describe the issue

Plugins page https://kestra.io/plugins is now showing sub-plugins, because of which we have multiple boxes having the same name "cli" which is appearing pretty weird.

![Screenshot 2024-06-08 at 5 05 30 PM](https://github.com/kestra-io/docs/assets/6594483/8095732b-756c-4fcd-87a4-518e9bd261d1)

Is including sub-plugins deliberate, or by mistake?
In case we are going ahead with sub-plugins, should the naming be made more specific like "Modal CLI", "SQLMesh CLI", "dbt CLI", etc.?

### Environment

- Kestra Version: 0.17.0
- Operating System (OS/Docker/Kubernetes): N/A
- Java Version (if you don't run kestra in Docker): N/A

**Comment by anna-geller** [comment]
Something must have gone wrong; it should look the same way as the Plugins UI in the application 😢 we'll investigate on Monday, thx so much, Shruti! <3

**Comment by anna-geller** [comment]
1. Blob should be called Azure Blob 
2. Dbt CLI shouldn't be called just CLI but Dbt CLI https://github.com/kestra-io/plugin-dbt/blob/9d7827b26322f9d10ee1707b9dc73f6686b243dd/src/main/java/io/kestra/plugin/dbt/cli/package-info.java#L2 
3. Bigquery should be called BigQuery https://github.com/kestra-io/plugin-gcp/blob/f8967288ee5514deadc0b64bc3054f86169caa4c/src/main/java/io/kestra/plugin/gcp/bigquery/package-info.java#L2 
4. Dbt shouldn't exist https://kestra.io/plugins?page=1&size=40&category=All+Categories&q=dbt 
5. ...etc

We should use the plugin subgroup's title

**Comment by anna-geller** [comment]
@Skraye my comment above shows that even though we specified Dbt CLI as title, it shows just as "Cli" https://github.com/kestra-io/plugin-dbt/blob/9d7827b26322f9d10ee1707b9dc73f6686b243dd/src/main/java/io/kestra/plugin/dbt/cli/package-info.java#L2 

![image](https://github.com/user-attachments/assets/6fcbde80-140d-46d3-8454-2765f85361a3)

there is some bug here that we don't use the title

**Comment by anna-geller** [comment]
@Skraye the issue is not fixed, can you have a look what went wrong?

![image](https://github.com/user-attachments/assets/6990a66a-4b68-49f7-8165-06e7cb2d9456)

**Comment by Skraye** [comment]
Have you checked that the API is up to date ?

**Comment by anna-geller** [comment]
nope, I only checked what's on the website, I assumed the website uses the latest API version after we released the fix last Friday? could you check?

**Comment by Skraye** [comment]
API wasn't updated, website displays the correct subgroup now, but they may still be plugins with subgroup not set.


![image](https://github.com/user-attachments/assets/76fcb29d-2654-4b11-b3c1-a60e6785d309)
